Bootstrap w/ pip

jakirkham opened the pull request:
To ensure we build `wincertstore` with `pip` as we do with other packages (including `certifi`) while avoiding a circular dependency (as `setuptools` depends on `wincertstore`), download the `pip` and `setuptools` wheels and use them to install `wincertstore`.

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

xref: https://github.com/conda-forge/certifi-feedstock/pull/47

cc @ocefpaf

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.


I do have some suggestions for making it better though...


For **recipe**:

 * License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).

jakirkham commented:
@conda-forge-admin, please re-render.

jakirkham committed:
Use SPDX compatible `license` metadata

jakirkham committed:
Fix dead link

Point to PyPI as the Bitbucket repo link doesn't work any more.

jakirkham committed:
MNT: Re-rendered with conda-build 3.20.3, conda-smithy 3.8.0, and conda-forge-pinning 2020.10.08.20.29.16

Now that the `license` follows SPDX and the `home` URL has been updated,
re-render to update other feedstock content accordingly.

jakirkham committed:
Bootstrap the install with `pip` and `setuptools`

To ensure we build `wincertstore` with `pip` as we do with other
packages (including `certifi`) while avoiding a circular dependency (as
`setuptools` depends on `wincertstore`), download the `pip` and
`setuptools` wheels and use them to install `wincertstore`.

jakirkham committed:
Bump build number to 1005

As we now build `wincertstore` with `pip`, bump the build number to
produce new packages.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jakirkham commented:
@conda-forge-admin, please re-render.
